ibc_proto/tests/unit.rs

```diff
@@ -1,6 +1,7 @@
-#[test]
-pub fn import_localhost() {
-    use ibc_proto::localhost::MsgCreateClient;
-    let x = MsgCreateClient { signer: vec![] };
-    assert_eq!(x.signer, vec![]);
-}
+// Commenting this out. Until proto files are stable does not make sense to add unit tests
```

Comment:
I wonder if we should keep updating the generated .rs files if the source .proto files are unstable.

Reply:
I think we should pin against a specific version of the Cosmos SDK: https://github.com/informalsystems/ibc-proto/issues/10